app/views/state_file/questions/submission_confirmation/edit.html.erb

```diff
@@ -23,9 +23,9 @@
   </p>
 
   <p>
-    <!-- TODO - replace with real link -->
-    <%= link_to t(".apply_to_state_benefits", state_name: States.name_for_key(params[:us_state].upcase)), "" %>
```

Comment:
It's not clear if we should keep this empty link or not, but it wasn't in the mock. I asked and can restore it if we need to keep it.